### PR TITLE
OpenWeatherMap support

### PR DESCRIPTION
**Pull request by werthdavid**
Grüße aus Stuttgart 😉

### CODE REVIEW

**Comment by naofireblade** [comment]
Thank you very much for your PR with the openweathermap api. That’s awesome!

I will look into this and merge it later today.

File: api/openweathermap.js [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+        'WindDirection',
+        'WindSpeed'
+    ],
+    forecastDays = 5;
+
+var debug, log;
+
+var init = function (apiKey, language, location, locationGeo, locationZip, locationCity, l, d) {
+    Openweathermap.setLang(language);
+    if (language) {
```
Comment (naofireblade) [inline api/openweathermap.js]: that looks like a typo

Comment (werthdavid) [reply api/openweathermap.js]: Ooops that was a copy&paste error, glad u saw it!

**Comment by naofireblade** [comment]
Give me one more day pls (:

**Comment by werthdavid** [comment]
not a problem!

File: api/openweathermap.js [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+var debug, log;
+
+var init = function (apiKey, language, location, locationGeo, locationZip, locationCity, l, d) {
+    Openweathermap.setLang(language);
+    if (location) {
+        Openweathermap.setCityId(location);
+    }
+    if (locationGeo) {
+        Openweathermap.setCoordinate(location[0], location[1]);
```
Comment (naofireblade) [inline api/openweathermap.js]: This should be locationGeo

**Comment by naofireblade** [comment]
Is there an easy way to work on your PR branch together? don't wanna bother you with typos :D .

**Comment by naofireblade** [comment]
Zip-Code is not working for me. And I have no option to set the country code that is requested by the api if you don't want to search for a us zip-code.

**Comment by naofireblade** [comment]
Also "Berlin" works despite giving more then one results here: https://openweathermap.org/find?q=berlin so I think the advice in the readme can be removed.

**Comment by werthdavid** [comment]
Agreed

**Comment by naofireblade** [comment]
I updated the readme a little bit, because I think forecast day and observation time/station should not be counted in the comparison table.

**Comment by werthdavid** [comment]
Fair enough

**Comment by naofireblade** [comment]
Published to npm in v2.2.0
https://www.npmjs.com/package/homebridge-weather-plus/v/2.2.0

**Pull request merged**